python/ray/autoscaler/aws/development-example.yaml

```diff
@@ -37,7 +37,19 @@ available_node_types:
         BlockDeviceMappings:
             - DeviceName: /dev/sda1
               Ebs:
-                  VolumeSize: 50
+                  VolumeSize: 250
+
+    quota_issue_worker:
```

Comment:
Is this just for testing? (Can it be removed from the final PR and added to the PR description instead?)

Reply:
Ok, the wuisawesome further down suggests yes :)
Don't forget to remove eventually.